docs/DeveloperGuide.md

```diff
@@ -927,3 +927,70 @@ long inputs, whether it be for `n/[NAME]` or `d/[DESCRIPTION]` or `t/[TAG]` caus
+- The new command will look something like `edit INDEX I/Index sub-n/Name`, which edits a task's subsection name. Edit by subsection description command can look like
+  `edit INDEX I/Index sub-d/Description`. Here, the first index corresponds to the main task's index in the current task list displayed and the index after
+prefix `I/` corresponds to the index of the subsection in the parent task.
+- The result of the command will be a displayed list of tasks found:
+  ![text-wrap0](images/edit-subsection.png)
+
+
+### 7.4 Enabling `schedule` by subsection
+
+- The application currently does not support scheduling subsections so that they can be completed on different days.
+#### Possible Solution
+- One possible solution is to allocate the main task's effort level to the subsection and allow the subsection to be one of a simple task, deadline or event. The scheduling algorithm will allocate big tasks by putting them across different days with different subsections on each day.
+- The new `subsection` command will look something like `subsection n/homework d/cs2109s [E/20] [D/DEADLINE] [F/FROM DATE] [T/TO DATE]`. If the effort level is not specified, then the effort level of the main task will be evenly distributed among the subsections.
+- The `schedule` command will still look like `schedule E/EFFORT D/DATE` but the result will be the parent tasks along with only part of the subsections that is set to be completed on that day.
+- For example, if the task list only contains a parent task that has an effort level of 50, with two subsections called "homework A" and "homework B" with 25 effort level each, when the user enters `schedule E/25 D/2023-05-04` and today is 2023-05-04, the displayed result 
+will only be the parent task and the subsection "homework A". If the user enters `schedule E/25 D/2023-05-05` then, the same parent task with subsection "homework B" will be displayed.
+- This method ensures that each subsection is treated as a somewhat individual task when scheduling, so that the main task's workload can be spread out. The scheduling algorithm will allocate the deadlines' subsections before the deadline date, even if that may result in 
+the total effort on a certain date exceeding daily effort level. Also, the events' subsections should occur between the events' start and end time. 
+- Moreover, allowing parent tasks to have deadlines and events as subsections will allow deadlines to be broken down into multiple deadlines and events to contain multiple events. Those components will be scheduled individually and displayed along with the parent tasks. 
+This ensures that the user have more flexibility in handling subsections.
+
+  
+### 7.4 Enabling `sort` by subsection
+
+- The application currently does not support sorting subsections so that they can be completed on different days.
```

Comment:
I'm not sure what you mean here, the sorting of subsections does not help support the completion on different days. You can just end the sentence at `The application currently does not support sorting subsections.` I think